CMakeLists.txt

```diff
@@ -287,16 +287,6 @@ if (ONEDPL_BACKEND MATCHES "^(tbb|dpcpp|dpcpp_only)$")
             endif()
         endif()
 
-        if (DEFINED ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION)
-            if(ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION)
-                message(STATUS "Adding -DONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION=1 option")
-                target_compile_options(oneDPL INTERFACE "-DONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION=1")
-            else()
-                message(STATUS "Adding -DONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION=0 option")
-                target_compile_options(oneDPL INTERFACE "-DONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION=0")
-            endif()
-        endif()
-
```

Comment:
Are these removals of `ONEDPL_WORKAROUND_FOR_IGPU_64BIT_REDUCTION` intentional?  It seems like this may be a rebase/merge gone wrong.

Reply:
Thank you, @danhoeflinger 
It's simple rebase / merge error, will fix it.

Reply:
Fixed.